### PR TITLE
chore(examples): Adding Env Dev Automatic Provisioning

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds a full example recipe on how to automate an environment provisioning from a script using organization tokens

## Changes

- Adds recipe and examples

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+For Tests and Test Suites, you can find the YAML definition by clicking the `Automate` tab.
+
+![Test Resource Definition](../img/test.gif)
+
+## Requirements
+
+**Tracetest CLI**:
+
+- Download & Install the [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference)
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx:61]:
```suggestion
- Download & Install the [Tracetest CLI](https://docs.tracetest.io/cli/cli-installation-reference).
```

File: docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+- Login or Signup to your Tracetest account using the CLI with `tracetest configure`
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx:62]:
```suggestion
- Login or Signup to your Tracetest account using the CLI with `tracetest configure`.
```

File: docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+**Docker Compose**:
+
+- Install [Docker Compose](https://docs.docker.com/compose/install/)
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx:66]:
```suggestion
- Install [Docker Compose](https://docs.docker.com/compose/install/).
```

File: docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+- Ensure Docker Compose is running on your machine
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx:67]:
```suggestion
- Ensure Docker Compose is running on your machine.
```

File: docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+
+## Run This Quckstart Example
+
+The example below is provided as part of the Tracetest GitHub repo. You can download and run the example by following these steps:
+Clone the Tracetest project and go to the Privisioning Developer Environment with CLI example directory:
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/provisioning-developer-environment-script.mdx:72]:
```suggestion
Clone the Tracetest project and go to the Provisioning Developer Environment with CLI example directory:
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by adnanrahic — APPROVED** [review]

**Review by kdhamric — APPROVED** [review]

**Pull request merged**